04/app.js

```diff
@@ -1,9 +1,46 @@
+// 1st solution
+function checkBirthday() {
+    const date = new Date();
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const dateToday = day + '/' + month;
+
+    // DD/MM
+    const userBorn = Object.values(this.born).join('/').slice(0, -5);
+    const userName = this.firstName;
+
+    output(dateToday, userBorn, userName);
+}
+
+// 2nd solution
+function checkBirthday2() {
+    const date = new Date();
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const dateToday = day + '/' + month;
+    const userBorn = this.born.day + '/' + this.born.month;
+    const userName = this.firstName;
+
+    return output(dateToday, userBorn, userName);
+}
+
+// Output message
+const output = function (date, born, name) {
+    if (date === born) {
```

Comment:
W zasadzie oba podejścia są podobne. Ja bym zrobił jeszcze inaczej - po prostu porównał osobno dzień oraz miesiąc - może byłoby bardziej czytelniej bo dla niektórych poprawna data to `22/03`, a dla innych `22-03`. Nie jestem pewny, ale może wyszłoby jakieś zamieszanie z tej formy. Myślę, że znów powinieneś upraszać, zamiast mocno eksperymentować ;P

Reply:
@devmentor-pl, wow – zmieniłem kod i to okazało się nawet szybsze i prostsze do napisania. Jednak zamiast porównania osobno dnia i miesiąca postawiłem na użycie logicznego "and".  Zdecydowałem się również na zamianę wszystkich porównywanych wartości na string, na wszelki wypadek, gdyby w którymś momencie dane uległy zmianie. 
Dziękuję bardzo!